README

```diff
@@ -108,6 +108,10 @@ Mandriva:
 	Mandriva has integrated fio into their package repository, so installing
 	on that distro should be as easy as typing ``urpmi fio``.
 
+Arch Linux:
+        Arch Linux package is provided under community.
```

Comment:
Just checked, and Arch already has version 2.21. Those guys move quick! I wish other distros were better at keeping uptodate.